support/main/default/customMetadata/Trigger_Object.Account.md-meta.xml

```diff
@@ -6,4 +6,8 @@
         <field>Enabled__c</field>
```

Comment:
below here is just update from things that I think were missed from renaming `TriggerHandler` -> `Manager`